src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Decentralized finance (defi)
```

Comment:
Consistency question: `defi` or `DeFi`? Looks like most places have adopted `DeFi`, judging from a quick google of "defi".

Reply:
I'm really curious everyones thoughts on this too. Same with dapp/Dapp/DApp (DAppNode for example uses this capitalization). Seems like we've been going with the lowercase approach which I'm not against, as it seems less like a specific product of some kind. That being said, I certainly have seen a lot of DeFi, but that could just be from time on DeFi Pulse

Reply:
have switched it to DeFi